### PR TITLE
MODCAL-87 Upgrade to RMB v32

### PR DESCRIPTION
**Pull request by alexanderkurash**
Resolves [MODCAL-87](https://issues.folio.org/browse/MODCAL-87)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,8 +24,8 @@
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-    <raml-module-builder.version>31.1.5</raml-module-builder.version>
-    <vertx.version>3.9.4</vertx.version>
+    <raml-module-builder.version>32.1.0</raml-module-builder.version>
```
Comment (OleksandrVidinieiev) [inline pom.xml:27]: As per Adam Dickmeiss in #releases:
> RMB 32.2.0 available. If your module is not updated to RMB 32 already, we suggest you use this version.

File: src/main/java/org/folio/rest/impl/CalendarAPI.java [hunk 2]
```diff
@@ -65,19 +68,16 @@
...
 import io.vertx.core.Promise;
-import io.vertx.core.logging.Logger;
-import io.vertx.core.logging.LoggerFactory;
 
 
 @SuppressWarnings({"squid:S1854", "squid:S1481"})
 public class CalendarAPI implements Calendar {
 
-  private static final Logger logger = LoggerFactory.getLogger(CalendarAPI.class);
+  private static final Logger logger = LogManager.getLogger(CalendarAPI.class);
```
Comment (OleksandrVidinieiev) [inline src/main/java/org/folio/rest/impl/CalendarAPI.java:80]: Should we try initializing the logger with no arguments?

> using getLogger() without argument requires Multi-Release for stack walking

We do have Multi-Release enabled in POM.
https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-320

File: src/test/java/org/folio/rest/impl/EmbeddedPostgresBase.java [hunk 3]
```diff
@@ -1,12 +1,30 @@
...
 import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.tools.PomReader;
+
+import io.vertx.core.buffer.Buffer;
+import io.vertx.ext.web.client.HttpResponse;
 
 public class EmbeddedPostgresBase {
+  private static final Logger LOGGER = LogManager.getLogger(EmbeddedPostgresBase.class);
+
+  public static final int GET_TENANT_DELAY_MS = 1000;
```
Comment (OleksandrVidinieiev) [inline src/test/java/org/folio/rest/impl/EmbeddedPostgresBase.java:23]: Is it a delay or a timeout?

File: src/test/java/org/folio/rest/impl/CalendarIT.java [hunk 4]
```diff
@@ -74,32 +79,30 @@
 
   @BeforeClass
   public static void setup(TestContext context) {
-    vertx = Vertx.vertx();
+    vertx = Vertx.vertx(new VertxOptions()
+      .setMaxEventLoopExecuteTimeUnit(TimeUnit.MILLISECONDS)
+      .setMaxEventLoopExecuteTime(GET_TENANT_DELAY_MS));
```
Comment (OleksandrVidinieiev) [inline src/test/java/org/folio/rest/impl/CalendarIT.java:84]: Where does this come from?

**Review by roman-barannyk — APPROVED** [review]

**Pull request merged**